text/rustconf-steering-committee.md

```diff
@@ -5,46 +5,49 @@
+- "Steering Committee": A group of advisors that present recommendations to an organizing group (in this case, the "organizing group" being the Rust Foundation as the organizer of RustConf 2024). Often, a steering committee will be in place to represent the interests of a specific group (in this case, the Rust Project and community)
+- "Program Committee": The group responsible for selecting speakers/talks to appear on the RustConf agenda/"program". The Program Committee for RustConf has historically been independent of the organizer and composed largely of members of the Rust Project. The Foundation intends to maintain this tradition in 2024 but will provide oversight and facilitation.
+- "CFP": Stands for "Call for Proposals". This is the period and process during which we will solicit talk submissions for RustConf. The Program Committee will select submissions from the CFP to appear on the program and might also advocate for inviting several speakers who did not submit a talk. 
+  
+# Proposal
 
-A group of ~5 volunteers from within the Leadership Council and the Project Directors responsible for providing input and direction to the Rust Foundation staff team regarding RustConf 2024. The Rust Foundation will lean on the steering committee to advocate for areas of desired involvement besides the traditional UnConference Day and Program Committee. 
+The Rust Foundation staff team is proposing the formation of a RustConf Steering Committee with the following parameters:
+- Members: 4 total members from the Rust Project -- one leadership council member, one Project Director member, and two members from the wider Rust community. We feel that an even split between leadership members and community members makes sense as the former positions can provide context and Project authority while the latter can advocate for those who are not necessarily in a position of power within the Project. We will lean on the Leadership Council as a whole to select this group based on a process of self-nomination initiated by the Foundation and promoted by the Leadership Council.  
+- Scope: The Foundation would like the Steering Committee's involvement in the following areas:
+ > - **Program Committee:** The Steering Committee should establish the structure, personnel, and processes of the Program Committee with oversight and collaboration from the Rust Foundation. The Foundation will take this plan from the Steering Committee to formally set up the Program Committee which will select talks after the closure of the call for talk proposals. Steering Committee members will not be on the Program Committee or be involved in selecting talks, although they can advise the Program Committee on key priorities as talks are being selected. To stay on schedule, the Foundation will likely need to open the CFP earlier than these two committees are in place, however, we will share our intentions for talk tracks/suggested topics with the Leadership Council in advance for feedback. 
```

Comment:
> with oversight and collaboration from the Rust Foundation

What are the limits to this oversight and collaboration. Is the Foundation allowed to disband the Steering Committee? Is the Steering Committee making proposals that the Foundation Staff ultimately has the right to give their approval or disapproval of?